cabal.project

```diff
@@ -96,3 +98,17 @@ benchmarks: true
 
 -- The only sensible test display option
 test-show-details: streaming
+
+
+constraints:
+  -- The API has changed for version 2.2, and there is no intention to change to that
+  -- version until after Conway.
+  , cardano-crypto-class ^>= 2.1
```

Comment:
Why is this necessary? API have changed in `cardano-crypto-class` current version on `master`, but the version 2.2 has not been releases, so this kind of speculative bounds should not exist in any `cabal.project` file.
```suggestion
```

Reply:
Removed. Pretty certain I added that when I was using an SRP for `cardano-base`.